lib/event_kind.dart

```diff
@@ -63,6 +63,8 @@ class EventKind {
 
   static const int GROUP_LEAVE = 9022;
 
+  static const int GROUP_CREATE_INVITE = 9009;
```

Comment:
Would you mind putting this up above GROUP_JOIN so it's in numerical order? Thanks!

Reply:
done